### ISSUE TITLE
nccs doesn't pick up SupportedStandards attribute from base class

### ISSUE THREAD

**Issue by devhawk** [neo-project/neo-devpack-dotnet#727]
smart contract framework has a very useful `Nep17Token` class that developers can inherit from to build their own token in a very small amount of code. The abstract `Nep17Token` class carries the `[SupportedStandards(new string[] { "NEP-17" })]` attribute. However, the class that inherits from `Nep17Token` will not have NEP-17 in the `supportedstandards` field of the manifest unless the concrete token subclass also has the `SupportedStandards` attribute

**Comment by erikzhang** [comment]
https://github.com/neo-project/neo-devpack-dotnet/blob/e1dae8f1f2ee066134d9e83539539c5c3fab19ec/src/Neo.Compiler.CSharp/CompilationContext.cs#L403

Normally, it should be able to obtain the attributes from the base classes.

**Comment by devhawk** [comment]
> Normally, it should be able to obtain the attributes from the base classes.

I'm not sure why it didn't work, but I just tried it w/ nccs 3.1.

Code:
``` cs
public class DevlogToken : Nep17Token
{
    public override string Symbol()
    {
        return "DVLGTKN";
    }
    
    public override byte Decimals()
    {
        return 8;
    }
}
```

manifest: 
``` json
{
    "name": "DevlogToken",
    "groups": [],
    "features": {},
    "supportedstandards": [],
// remaining manifest omitted
```

**Comment by Liaojinghui** [comment]
Well, it works fine in my machine

```json
"supportedstandards":["NEP-11","NEP-17"]
```

**Comment by erikzhang** [comment]
@devhawk Did you test the master branch? 3.1.0 does not support it.

**Comment by devhawk** [comment]
> @devhawk Did you test the master branch? 3.1.0 does not support it.

I did not. Will try with 3.2 when it ships

**Comment by erikzhang** [comment]
Fixed by #705
